react-native/react/reducers/login2.mobile.js

```diff
@@ -59,153 +64,124 @@ const initialState: LoginState = {
   forgotPasswordEmailAddress: '',
   forgotPasswordSubmitting: false,
   forgotPasswordSuccess: false,
-  forgotPasswordError: null
+  forgotPasswordError: null,
+  userPass: {
+    username: '',
+    passphrase: ''
+  },
+  deviceName: {
+    onSubmit: () => {},
+    existingDevices: [],
```

Comment:
we only show you this screen when you actually get the list so it's not possible to be in the 'not fetched' state